### DISCUSSION TITLE
Disable password login when using OIDC or LDAP

### DISCUSSION THREAD

**Discussion by wuast94** [mealie-recipes/mealie#4581] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

ATM its only possible to disable the registration but not the Login via username and password when using SSO

### Please provide a concise description of the feature that would resolve your issue.

an ENV to disable login via username/password

### Please consider and list out some caveats or tradeoffs made in your design decision

no

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by cmintey** [comment]
You can disable credentials login for specific users by setting their Auth Method to `OIDC` in the admin page. Also if users are originally created with an OIDC login, then this is the default

**Reply by wuast94** [reply]
thats when they are already authentikated

![image](https://github.com/user-attachments/assets/8a13aa43-6732-43df-b7f5-3280bf8d5492)

i would like to disable the username and password part from the login page, so its completly removed.

**Reply by cdanis** [reply]
#4943 is a draft of a setting for this.

**Comment by cdanis** [comment]
I'm also interested in this feature, and might write a PR.
